flow/compositor_context.h

```diff
@@ -77,8 +77,8 @@ class FrameDamage {
   }
 
   // Calculates clip rect for current rasterization. This is diff of layer tree
-  // and previous layer tree + any additional provideddamage.
-  // If previous layer tree is not specified, clip rect will be nulloptional,
+  // and previous layer tree + any additional provided damage.
+  // If previous layer tree is not specified, clip rect will be null,
```

Comment:
This one is referencing `std::nullopt`, not `null`.  You can leave as-is or change to `nullopt` or `std::nullopt`
```suggestion
  // If previous layer tree is not specified, clip rect will be nullopt,
```

Reply:
wow nullopt - interesting thing to learn today